changelog.d/15395.misc

```diff
@@ -0,0 +1 @@
+Implement [MSC2174](https://github.com/matrix-org/matrix-spec-proposals/pull/2174) to move the `redacts` key to a `content` property.
```

Comment:
Note that MSC2174 says to redact this property, but since the implementation is pair with MSC2176 which states that the entire body of the create event is to be preserved I think we're OK here since we're not adding it to the body anyway?

Reply:
This comment thread would likely have been more useful on the `msc2176_redaction_rules` variable in `RoomVersion` -- I think my point was that this is essentially paired with MSC2176 in implementation. You can't enable just this MSC.